### DISCUSSION TITLE
How to ignore python warning E501 (Line too long)

### DISCUSSION THREAD

**Discussion by dirkesquire** [prabirshrestha/vim-lsp#1528] [Q&A]
I have a wide screen and do not wish my code to truncate after 80 characters.

How can I tell it to ignore "pycodestyle:Warning:E501:E501 line too long (112 > 79 characters)"?

Thank you!

**Comment by dirkesquire** [comment]
If it helps I know one can pass in an argument to flake8 like this:
`--ignore=E501`

I'm just not how to do that here.

**Comment by lnxwrld** [comment]
find the file "flake8/defaults.py" and edit it , change MAX_LINE_LENGTH = 79 to some large number

**Comment by miguel9554** [comment]
This fixed it for me:

```vim
  let g:lsp_settings = {
  \  'pylsp-all': {
  \    'workspace_config': {
  \      'pylsp': {
  \        'plugins': {
  \          'pycodestyle': {
  \            'ignore': ['E501', 'E241'],
  \          },
  \          'flake8': {
  \            'ignore': ['E501'],
  \          },
  \        },
  \      },
  \    },
  \  },
  \}
```
